test/conftest.py

```diff
@@ -80,6 +80,41 @@
 DC_API_KEY = "NO_KEY"
 MOCK_DC = True
 
+# Documents collections
+DOCS_ALL_FORMATS = [
+    # Document object for a doc
+    Document(
+        content="My name is Christelle and I live in Paris",
+        meta={"meta_field": "test3", "name": "filename3", "date_field": "2018-10-01", "numeric_field": 4.5},
+    ),
+    Document(
+        content="My name is Camila and I live in Madrid",
+        meta={"meta_field": "test4", "name": "filename4", "date_field": "2021-02-01", "numeric_field": 3.0},
+    ),
+    Document(
+        content="My name is Matteo and I live in Rome",
+        meta={"meta_field": "test5", "name": "filename5", "date_field": "2019-01-01", "numeric_field": 0.0},
+    ),
+    # "dict" format
+    {
+        "content": "My name is Carla and I live in Berlin",
+        "meta": {"meta_field": "test1", "name": "filename1", "date_field": "2020-03-01", "numeric_field": 5.5},
+    },
+    # metafield at the top level for backward compatibility
+    {
+        "content": "My name is Paul and I live in New York",
+        "meta_field": "test2",
+        "name": "filename2",
+        "date_field": "2019-10-01",
+        "numeric_field": 5.0,
+    },
+]
+DOCS = [Document.from_dict(doc) for doc in DOCS_ALL_FORMATS]
+
+# Other constants for quicker integration tests
+SMALL_READER_MODEL = "distilbert-base-uncased-distilled-squad"
```

Comment:
deepset/tinyroberta-squad2 is our own distilled model of the same size. You could try that one. Should achieve the same speed but might have better prediction quality.